analyzer/server.go

```diff
@@ -179,14 +233,19 @@ func NewServer(port int, router *mux.Router) (*Server, error) {
 
 	pipeline := mappings.NewFlowMappingPipeline([]mappings.FlowEnhancer{gfe})
 
+	flowtable := flow.NewFlowTable()
+
 	server := &Server{
 		Port:                port,
 		Router:              router,
 		TopoServer:          tserver,
 		GraphServer:         gserver,
 		FlowMappingPipeline: pipeline,
+		FlowTable:           flowtable,
 	}
+	server.RegisterStaticEndpoints()
 	server.RegisterRpcEndpoints()
+	go flowtable.AsyncExpire(server.flowExpire, 10*time.Minute)
```

Comment:
I do not the purpose of having a callback function (flowExpire) which is only using the flowtable remove function. It seems to me this is an useless indirection ?